Makefile

```diff
@@ -1,62 +1,39 @@
-CORE_VERSION := 2017.4
+QIIME2_RELEASE := 2017.4
 HOSTNAME := qiime2core2017-4
 
 .PHONY: help
 help:
 	@echo "Please use \`make <target>' where <target> is one of"
 	@echo "  docker            to build a Docker image"
+	@echo "  vbox-bootstrap    to prep a Virtualbox OVF"
+	@echo "  vbox              to build a Virtualbox image"
 	@echo "  aws               to build an AWS image"
-	@echo "  virtualbox        to build a Virtualbox image"
-	@echo "  workshop-deploy   to build and deploy a workshop cluster"
-	@echo "  workshop-destroy  to tear down a workshop cluster"
 
 .PHONY: docker
 docker:
-	packer build \
-		-only=docker \
-		-var-file=variables.json \
-		qiime2core.json
-	docker run -i -t -d --name q2 qiime2/core:latest /bin/bash
-	docker commit \
-		-c 'ENV PATH=/miniconda3/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin' \
-		-c 'ENV LC_ALL=C.UTF-8' \
-		-c 'ENV LANG=C.UTF-8' \
-		-c 'VOLUME ["/data"]' \
-		-c 'WORKDIR /data' \
-		q2 qiime2/core:latest
-	docker stop q2 && docker rm q2 && docker tag qiime2/core:latest qiime2/core:$(CORE_VERSION)
+	docker build \
+		-t qiime2/core:$(QIIME2_RELEASE) \
+		--build-arg QIIME2_RELEASE=$(QIIME2_RELEASE) .
 
-.PHONY: aws
-aws:
+.PHONY: vbox-bootstrap
+vbox-bootstrap:
 	packer build \
-		-only=amazon-ebs \
-		-var-file=variables.json \
-		-var 'core_version=$(CORE_VERSION)' \
-		-var 'hostname=$(HOSTNAME)' \
-		qiime2core.json
+		-var 'QIIME2_RELEASE=$(QIIME2_RELEASE)' \
+		-var 'HOSTNAME=$(HOSTNAME)' \
+		-on-error=ask \
+		packer_vars/vbox-bootstrap.json
 
-.PHONY: virtualbox
-virtualbox:
+vbox: output-virtualbox-iso/QIIME_2_BASE_IMAGE.ovf
```

Comment:
Does this work? I'm not sure if Make will be able to find the relevant prereq target (`vbox-bootstrap`) if that file is outdated.